src/ngraph/frontend/onnx_import/ops_bridge.cpp

```diff
@@ -109,6 +109,11 @@ namespace ngraph
                     return ops_bridge::get()(node);
                 }
 
+                static bool is_op_type_supported(const std::string& op_type)
```

Comment:
This function may also be `const` qualified.

Reply:
Static member function cannot have `const` qualifier.

Reply:
Good point! :)